doc/sphinx-guides/source/container/dev-usage.rst

```diff
@@ -0,0 +1,49 @@
+Development Usage
+=================
+
+Please note! This Docker setup is not for production!
```

Comment:
If it cannot be used in production, please provide some insight why that is not a good idea.

Reply:
I disagree. This section has a heading "Development Usage" and is targeting someone hacking on Dataverse and who wants to testrun changes in containers rather than a local installation of all components.

Reply:
Please note! This Docker setup is not for production! 😄 

We could add, "You're in the dev-usage doc!" 😄 

We are deliberately walking before we run. 🚶 🏃